engine/src/main/java/org/entando/entando/aps/servlet/ExceptionHandlerServlet.java

```diff
@@ -0,0 +1,64 @@
+package org.entando.entando.aps.servlet;
+
+import com.agiletec.aps.system.services.lang.ILangManager;
+import com.agiletec.aps.system.services.lang.Lang;
+import com.agiletec.aps.system.services.page.IPage;
+import com.agiletec.aps.system.services.page.IPageManager;
+import com.agiletec.aps.system.services.url.IURLManager;
+import java.io.IOException;
+import java.util.Map;
+import javax.servlet.ServletConfig;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.context.support.SpringBeanAutowiringSupport;
+
+@Slf4j
+public class ExceptionHandlerServlet extends HttpServlet {
+
+    @Autowired
+    private IPageManager pageManager;
+    @Autowired
+    private ILangManager langManager;
+    @Autowired
+    private IURLManager urlManager;
+
+    @Override
+    public void init(ServletConfig config) throws ServletException {
+        super.init(config);
+        SpringBeanAutowiringSupport.processInjectionBasedOnServletContext(this, config.getServletContext());
+    }
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response)
+            throws ServletException, IOException {
+        try {
+            String pageCode = this.pageManager.getConfig(IPageManager.CONFIG_PARAM_ERROR_PAGE_CODE);
```

Comment:
pls add more log to debug to understand why is not loading custom error page.

Reply:
done